### PR TITLE
fixes #145, restructures README and adds test command

### LINKED ISSUE #145: Documentation - Update Getting Started
**Is your feature request related to a problem? Please describe.**
Our Readme should have a section with local setup details. 

**Describe the solution you'd like**
- In Readme, add testing under the `Getting Started` section
- Ensure the Publishing section is *not* under Getting started.

**Testing notes**
[Tester to complete]

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #145

adds test note and restructures headings

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ cd psammead && npm install
 
 NB, this automatically runs `npm run install:packages` in a postinstall step for you.
 
+### Run tests
```
Comment (dr3) [inline README.md:19]: Not sure if this is in scope of this issue, but we should write about how testing is run at a repo level, and its is not the responsibility of the package to setup testing

File: README.md [hunk 2]
```diff
@@ -28,21 +34,21 @@ npm run storybook
...
 
-#### Publishing a package
+### Publishing a package
 
 - In your terminal, run `npm login` and follow the steps for authentication
 - `npm publish packages/<PATH_TO_PACKAGE> --otp=<YOUR_2FA_CODE>`
   e.g. `npm publish packages/components/psammead-paragraph --otp=<YOUR_2FA_CODE>`
 
-#### Making a package public
+### Making a package public
```
Comment (dr3) [inline README.md:56]: This can be done in a single command with `--access public`, might be simpler

**Review by dr3 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ChrisBAshton) [reply README.md:19]: Sure thing - thanks for the feedback. Will do now.

File: README.md (hunk 2 above)
Comment (ChrisBAshton) [reply README.md:56]: Hmm agreed this command is simpler, but then we'd have to add a line about `cd`ing into the package directory, whereas the current command works from the root level (as does the one above that). So for consistency I'd suggest we stick with what we have.

Comment (dr3) [reply README.md:56]: Oh no soz I meant doing 
```
npm publish packages/components/psammead-paragraph --access public --otp=<YOUR_2FA_CODE>
```
Not overly bothered though

**Review by dr3 — APPROVED** [review]

**Review by radiocontrolled — APPROVED** [review]

**Pull request merged**